compiler/src/dotty/tools/dotc/ast/Trees.scala

```diff
@@ -579,7 +579,7 @@ object Trees {
   }
 
   /** Array(elems) */
-  class JavaSeqLiteral[T >: Untyped] private[ast] (elems: List[Tree[T]], elemtpt: Tree[T])(implicit @constructorOnly src: SourceFile)
+  class JavaSeqLiteral[-T >: Untyped] private[ast] (elems: List[Tree[T]], elemtpt: Tree[T])(implicit @constructorOnly src: SourceFile)
```

Comment:
This is good for uniformity but why would this make any difference for type test safety warnings ?

Reply:
The following code shows why it matters:

```Scala
trait Tree[-T]

class JavaSeqLiteral[T] extends Tree[T]

trait Type

class DummyTree extends JavaSeqLiteral[Any]

def foo1(tree: Tree[Type]) =
  tree.isInstanceOf[JavaSeqLiteral[Type]]   // error

foo1(new DummyTree)
```